exporter/views.py

```diff
@@ -39,6 +49,14 @@ def wiper_start(request):
 
 @csrf_exempt
 def exporter_status(request):
+    """
+    Returns the status of (potentionaly) running export job.
+
+    Expects {"spider": <spider>, "job_id": job_id} in request body.
+
+    Returns {"status": "ok", "data": <status>}
+    where status is one of WAITING, RUNNING, COMPLETED.
```

Comment:
Please use job vs task correctly. Otherwise, it is confusing.

```suggestion
    Returns the status of an exporter job task.

    Expects ``{"spider": <spider>, "job_id": <job_id>}`` in the request body.

    Returns ``{"status": "ok", "data": <status>}`` where status is one of WAITING, RUNNING, COMPLETED.
```

Reply:
cbom also sends collection_id, though the `exporter_status` handler doesn't seem to care.